posts/2021-08-18_dingwall_to_cromarty/content.html

```diff
@@ -0,0 +1,50 @@
+  
+<p><img src="img/image04.jpg" /></p>  
+<p><img src="img/image05.jpg" /></p> 
+  
+<p>We made it to Cromarty at low tide, and could look back on a successful trip, helped by the wind and tidal conditions. Cromarty has a dutch cheese store, so while we started the trip with some Canadian interaction, we ended with some Dutch. A fitting trip for us. As we snacked on some treats from the Cromarty bakery we watched an oil platform being towed out of the firth. Live entertainment!</p>
+
+<p><img src="img/image06.jpg" /></p> 
+
+<h3><strong>B</strong>y public transport (2 hours)/></h3>
+
+<p>After the paddle, it was time to go back home. But, our car was in Dingwall. Oh those fun shuttle arrangements when on a kayak trip. The bus timetables indicated a bus between Cromarty and Dingwall does exist, but the timetables proved too complex for us to understand. Google maps’ public transport section indicated that the bus would bring you back to Inverness, from where you take the bus to Dingwall. We decided to take the bus to where it goes left to Inverness and Dingwall would be to the right, where we had already stacked a bike in the morning while driving with the kayaks on the roof to Dingwall, to bike the last hour to Dingwall from there, instead of taking the detour to Inverness. The bus went through every village on the Black Isle and was full of school kids. The busdriver gave me a ticket for a kid as I discovered later, which I take it is due to me looking so young, but it could also been that he got so confused by where I wanted to get out of the bus that he forgot to switch the ticket machine to adult. The ride was long, and at one point I could even see our house on the other side of the Moray firth. But, eventually it stopped at my bike, and I could bike back to the car, which brings us to the next type of transport. </p>
```

Comment:
```suggestion
<p>After the paddle, it was time to go back home. But, our car was in Dingwall. Oh, those fun shuttle arrangements when on a kayak trip. The bus timetables indicated a bus between Cromarty and Dingwall does exist, but the timetables proved too complex for us to understand. Google Maps’ public transport section indicated that the bus would bring you back to Inverness, from where you take the bus to Dingwall. We decided to take the bus to where it goes left to Inverness and Dingwall would be to the right. We had already stashed a bike in the morning while driving with the kayaks on the roof to Dingwall, intending to bike the last hour to Dingwall from there, instead of taking the detour to Inverness. The bus went through every village on the Black Isle and was full of school kids. The bus driver gave me a ticket for a kid as I discovered later, which I take it is due to me looking so young, but it could also been that he got so confused by where I wanted to get out of the bus that he forgot to switch the ticket machine to adult. The ride was long, and at one point I could even see our house on the other side of the Moray firth. But, eventually it stopped at my bike, and I could bike back to the car, which brings us to the next type of transport. </p>
```